src/components/QuranReader/TafsirView/TafsirVerseText/index.tsx

```diff
@@ -9,7 +9,7 @@ import { getVerseWords } from 'src/utils/verse';
 import Verse from 'types/Verse';
 
 interface Props {
-  verses: Record<string, Verse>;
+  verses: Verse[];
 }
 
 const TafsirVerseText: React.FC<Props> = ({ verses }) => {
```

Comment:
Since this is becoming a general-purpose component and not related to Tafisr anymore, we should rename it to maybe something like `VerseTextPreview` or something similar and move it up in the directory tree.